packages/teams-js/src/private/index.ts

```diff
@@ -18,20 +18,23 @@ export {
   registerUserSettingsChangeHandler,
   openFilePreview,
 } from './privateAPIs';
-export { conversations, OpenConversationRequest } from './conversations';
+export * as conversations from './conversations';
+//It is necessary to export OpenConversationRequest from conversations.ts individually as well
```

Comment:
What's special about OpenConversationRequest that we have to export it individually?

Reply:
Prior to this change, it was listed as a named export, so anyone could have been referencing it as `import { OpenConversationRequest} from '@microsoft/teams-js'`. If we only had the `export * as conversations` line exporting, they would have to reference it as `conversations.OpenConversationRequest`, which would be a breaking change.